svr-tcpfwd.c

```diff
@@ -210,8 +216,16 @@ static int svr_remotetcpreq() {
 		 * has to remember it if it's to be cancelled */
 		m_free(request_addr);
 		m_free(tcpinfo);
+	} else {
```

Comment:
Can you move this bit this above "out:" ?

Don't think logging should be turned on generally. If you have a particular use case maybe make it a options.h option.